gadgets/bpfstats/README.mdx

```diff
@@ -0,0 +1,314 @@
+                                                         737 ig_execve_x                   160239             317          837504      1
+                                                         738 ig_fa_pick_e                  373093             509           21632      2
+                                                         739 ig_fa_pick_x                  378858             509         1194336      4
+                                                         740 ig_sched_exec                 112317              67          837504      1
+a624bdbf27f mytraceexec          trace_exec:%IG_TAG%     743 security_bprm_c                49435              67         8372040      4
+a624bdbf27f mytraceexec          trace_exec:%IG_TAG%     744 ig_execve_e                   140296             317         8372040      4
+a624bdbf27f mytraceexec          trace_exec:%IG_TAG%     745 ig_execveat_e                      0               0         8372040      4
+a624bdbf27f mytraceexec          trace_exec:%IG_TAG%     746 ig_execve_x                    89667             317         8647840      5
+a624bdbf27f mytraceexec          trace_exec:%IG_TAG%     747 ig_sched_exec                  76171              67         8647840      5
+a624bdbf27f mytraceexec          trace_exec:%IG_TAG%     748 ig_execveat_x                      0               0         8647840      5
+bb95fa2c8ab mytraceopen          trace_open:%IG_TAG%     749 ig_open_e                          0               0         1098240      3
+bb95fa2c8ab mytraceopen          trace_open:%IG_TAG%     750 ig_openat_e                 21538317           48300         1098240      3
+bb95fa2c8ab mytraceopen          trace_open:%IG_TAG%     751 ig_open_x                          0               0         1615904      4
+bb95fa2c8ab mytraceopen          trace_open:%IG_TAG%     752 ig_openat_x                  9076187           48503         1615904      4
+```
+  </TabItem>
+</Tabs>
+
+Finally, clean the system:
+
+<Tabs groupId="env">
+  <TabItem value="kubectl-gadget" label="kubectl gadget">
+```bash
+$ kubectl gadget remove mytraceexec mytraceopen
+```
+  </TabItem>
+
+  <TabItem value="ig-daemon" label="ig-daemon">
+```bash
+$ sudo gadgetctl remove mytraceexec mytraceopen
+```
+  </TabItem>
+</Tabs>
+
+### Exporting metrics
+
+The `bpfstats` Gadget provides the following metrics for `mapCount`,
+`mapMemory`, `runcount` and `runtime`. To enable the metrics listener, check the
+[Exporting Metrics](../reference/export-metrics.mdx) documentation. To enable
+the collector for this gadget, run the following command:
+
+<Tabs groupId="env">
+    <TabItem value="kubectl-gadget" label="kubectl gadget">
+        ```bash
+        WIP: Headless mode for kubectl gadget is under development
+        ```
+    </TabItem>
+
+    <TabItem value="ig" label="ig">
+        ```bash
+        $ sudo gadgetctl run bpfstats --otel-metrics-name=bpfstats:bpfstats --annotate=bpfstats:metrics.collect=true --detach
```

Comment:
Tested and worked properly. I suggest specifying a name to avoid using the id when deleting it.